tests/unit/configuration/test_paths.py

```
@@ -0,0 +1,120 @@
+            side_effect=Exception("boom"),
+        ):
+            result = delete_session_file("proj123", "sess456")
+            assert result is False
+
+class TestCheckForUpdates:
+    def test_returns_false_on_subprocess_failure(self):
+        with patch(
+            "tunacode.configuration.paths.subprocess.run",
+            side_effect=Exception("pip not found"),
+        ):
+            has_update, version = check_for_updates()
```

Ruff (0.14.14)

[warning] Unpacked variable `version` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)